### PR TITLE
add form and control for options

### PR DESCRIPTION
**Pull request by tektechnologies**
working through lab 4 and will be ready for review, by tonight @selmaT273

### CODE REVIEW

**Comment by tektechnologies** [comment]
My problem domain has changed from Favorite Foods to selecting themes from the environment. May have to refactor some of the function names.

**Comment by tektechnologies** [comment]
now I need to refactor the problem domain naming to reflect not only a list of Favorite Foods, but a list of Favorites worlds or paths or things?  Then I will move on the lab 5 . 
@selmaT273 I may also do bootstrap tomorrow on the site and possibly look at the mobile version since it is live......

**Comment by tektechnologies** [comment]
git is rendering the Pick your reaction in order of list and not order of picked, this is not relevant.

@selmaT273

**Comment by tektechnologies** [comment]
it is failing I think because of the comments I have in the code. will remove after review. @selmaT273 
It was not the comments it was the {} The comments were fine.

File: src/views/FoodMain.js [hunk 1]
```diff
@@ -2,9 +2,16 @@ import React from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import CardColumns from 'react-bootstrap/CardColumns';
 import FavoriteFoods from './FavoriteFoods';
-import { Form } from 'react-bootstrap/Form';
+import {Form}  from 'react-bootstrap';
 
+{/** So the above renders but this doesnt
+  import {Form}  from 'react-bootstrap/Form';
```
Comment (tektechnologies) [inline src/views/FoodMain.js]: @dahlbyk  any idea why this is?

Comment (selmaT273) [reply src/views/FoodMain.js]: Not sure why it is, but I typically haven't been putting { } around my bootstrap components I'm bringing in, although I haven't tried using Form yet.

Comment (tektechnologies) [reply src/views/FoodMain.js]: To write comments in JSX, you need to use JavaScript's forward-slash and asterisk syntax, enclosed inside a curly brace {/* comment here */}

I was in the wrong spot

Comment (tektechnologies) [reply src/views/FoodMain.js]: J to the S to the X

Comment (selmaT273) [reply src/views/FoodMain.js]: Sorry i should've been specific- i meant in response to your comment in the code about the import {Form}. I think you need { } if you're don't specify the react-bootstrap/Form path, but if you specify the /Form endpoint then you don't need the { }.

**Review by selmaT273 — APPROVED** [review]
Your dropdown menu (Form) is working! One thing I'd recommend is cleaning up the code, specifically your spacing and indentation. Keeping this in mind as you write out each piece of code can be helpful, not only for debugging but also makes it easier to read and understand. For example in the FavoriteImageMain, the event handlers appear to line up as if they're outside of the class.

**Pull request merged**